pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -429,16 +447,11 @@ func manageOpenShiftControllerManagerDeployment_v311_00_to_latest(client appscli
 				}
 				newEnvs = append(newEnvs, env)
 			}
-			// reflect.DeepEqual does not consider this case equal
```

Comment:
why is this no longer needed?

Reply:
because the new approach is to do a hash of the whole spec: https://github.com/openshift/library-go/blob/245d1ca16d1514313f3a7ab13ff2c9e3b07b5f43/pkg/operator/resource/resourceapply/apps.go#L95

and compare that here https://github.com/openshift/library-go/blob/master/pkg/operator/resource/resourceapply/apps.go#L131

Reply:
@knelasevero for awareness about this change. We need to do the same for https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/pkg/operator/target_config_reconciler.go#L591 and https://github.com/openshift/secondary-scheduler-operator/blob/master/pkg/operator/target_config_reconciler.go#L276-L277. Would you please take a look?

Reply:
Will do